content/guides/references/migration-guide.md

```diff
@@ -175,6 +175,188 @@ Cypress comes bundled with its own [Node.js version](https://github.com/cypress-
+
+Remember that from 7.0 you can configure `cypress.json` and have specific flags for component testing or e2e.
+
+For example if you want video recording enabled only for End-to-end testing, you can disbale it for compoent testing like so
+
+```json
+{
+  "testFiles": "**/*spec.{js,jsx,ts,tsx}",
+  "video": true,
+  "component": {
+    "componentFolder": "src/components",
+    "video": false
+  }
+}
+```
+
+[Read the docs for the new config.](/guides/references/configuration)
+
+### Plugins File
+
+#### Predefined React Plugins
+
+First, if you are using a predefined plugin of `@cypress/react`, Component Testing is still configured the same.
+You will not need to change anything to your configuration for component tests 7.0 to work.
+
+To know if you are using the predefined plugins file, look at the path imported/required in your plugins file.
+If the path contains `@cypress/react/plugins/...` you are using one.
+
+<Alert type="info">
+
+You are requiring from `cypress-react-unit-test/plugins/...`? Then you are using an old version of Cypress Component Testing. <br/>
+The package has changed name last year and has moved to the `@cypress` npm organisation.
+Please install `@cypress/react` and update the required path to `@cypress/react/plugins/...`
+
+</Alert>
+
+For example, this `cypress/plugins.js` file is still valid.
+
+```js
+const injectNextDevServer = require('@cypress/react/plugins/next')
+
+module.exports = (on, config) => {
+  injectNextDevServer(on, config)
+
+  return config
+}
+```
+
+#### Dev-Servers Instead of Preprocessors
+
+Before 7.0, Cypress Component testing used preprocessors to bundle your code before serving it to the cypress tests.
+You could have a plugins file like the following:
+
+<Badge type="danger">Before</Badge>
+
+```js
+const webpackPreprocessor = require('@cypress/webpack-preprocessor')
+const webpackConfig = require('../webpack.config')
+
+module.exports = (on, config) => {
+  on('file:preprocessor', webpackPreprocessor(options))
+
+  return config
+}
+```
+
+In 7.0 we introduced the concept of a test "dev-server". So you will have to change the plugins file like so.
+
+<Badge type="success">After</Badge>
+
+```js
+const { startDevServer } = require('@cypress/webpack-dev-server') // replace the preprocessor with a dev-server
+const webpackConfig = require('../webpack.config')
+
+module.exports = (on, config) => {
+  on('dev-server:start', (options) => {
+    // use the dev-server:start event instead of file preprocessor
+    return startDevServer({ options, webpackConfig })
+  })
+
+  return config
+}
+```
+
+### Support File
+
+Good news: You no longer require a support file.
+
+If you find the following line in your support file, remove it, it is not useful anymore.
+
+<Badge type="danger">Before</Badge>
+
+```js
+import '@cypress/react/hooks' // OR import '@cypress/react/hooks'
+```
+
+We kept the file in the published version to avoid breaking changes. But the file does not do anything.
```

Comment:
I don't understand this line at all. Many users will just have a support file, they use it for lots of other things. How are you controlling whether they have a support file? What it this published version?

Reply:
I meant that the "hooks" file has been deprecated. But I agree the sentence is very confusing.

Reply:
I will try to find a better formulation

Reply:
```suggestion
We kept the `hooks.js` file in version 7.0 to avoid a breaking change. But the hooks file does not do anything anything.
```